resolvers.py

```diff
@@ -19,20 +19,35 @@ def get_dates_and_price_check(
             _,
             info,
             **search_criteria
-    ) -> List[DatesCheck]:
-        validated_search_criteria = DatesAndPriceCheckInputValidator(**search_criteria)
-        id_houses_dict = {}
-        for listing_id in validated_search_criteria.listing_ids:
-            id_houses_dict[listing_id] = data[listing_id]
-        return DatesCheckGetter.get_date_check_results(validated_search_criteria, id_houses_dict)
+    ) -> List[Union[DatesCheck, Error]]:
+        try:
+            validated_search_criteria = DatesAndPriceCheckInputValidator(**search_criteria)
+            dates_and_price_check_results = []
+            for listing_id in validated_search_criteria.listing_ids:
+                if listing_id in data.keys():
+                    dates_and_price_check_results.append(
+                        DatesCheckGetter.get_date_check_result_for_house(
```

Comment:
`...dateS_check_...`
can you do a `cmd + shift + F` to see where else this typo is made?